db/migrate/20160114011126_create_indices.rb

```diff
@@ -0,0 +1,13 @@
+class CreateIndices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :indices do |t|
+      t.datetime :finalized_at
+      t.integer :category_id
```

Comment:
Well, I think overall that it better conveys the concept of what is being accessed by the relation. gLynx does a similar thing when providing specific relations for primary_organizations, funding_organizations, etc..

Code wise, the only impact of using `awrvi_version` as the `belongs_to` name and `awrvi_version_id` as the foreign key field is that you need to specify the class name on the belongs_to.